package.json

```diff
@@ -110,29 +110,29 @@
     "eslint_d": "^8.0.0",
     "extract-text-webpack-plugin": "^3.0.2",
     "flow-bin": "^0.122.0",
-    "flow-typed": "^2.5.1",
+    "flow-typed": "^3.1.0",
```

Comment:
Just to be sure, are we ok with this change? From the changelog it seems the only braking change is that they've dropped support for node < 10.

Reply:
@tiagoengel - Yeah I checked using `npm audit` and there was vulnerabilities in pre 3 and when upgrading they went away. Looks like the changelog didn't cover deep bumps

Reply:
My main concern is regarding to us breaking projects using node < 10, if that is not a concern than it's fine by me.

Reply:
This will be fine as all projects internally are on node 12 as the LTS, including Backpack so should be good here as this is a devDependency to Backpack only anyway so wouldn't have effect on anyone externally